### DISCUSSION TITLE
Brats transform float32 versus int8

### DISCUSSION THREAD

**Discussion by mioDio** [Project-MONAI/MONAI#1661] [General]
Hi,

I'm experimenting with your tutorials and the Brats Jupyter notebook contains the data transform called "ConvertToMultiChannelBasedOnBratsClassesd". I'm wondering why the transform is using np.float32
to convert the masks to the labels. Is it correct that each channel will only have 1s and thus it could just be a np.unit8?
Wouldn't that save considerable memory?

Happy to be corrected
Cheers

**Comment by Nic-Ma** [comment]
Hi @mioDio ,

Thanks for raising this ticket, @wyli already submitted PR https://github.com/Project-MONAI/MONAI/pull/1671 to update it.
Need @IsaacYangSLA  to help verify locally with Brats dataset.

Thanks.
